src/functionaltests/java/com/ericsson/ei/threadingAndWaitlistRepeat/ThreadingAndWaitlistRepeatSteps.java

```diff
@@ -79,7 +78,7 @@ public void no_event_is_aggregated() throws Throwable {
     public void the_waitlist_will_try_to_resent_the_events_at_given_time_interval() throws Throwable {
         TimeUnit.SECONDS.sleep(5);
         List<String> resentEvents = new ArrayList<String>();
-        List<String> lines = new ArrayList<String>(Files.readAllLines(Paths.get(LOGFILE)));
+        List<String> lines = new ArrayList<String>(Files.readAllLines(tempLogFile.toPath()));
```

Comment:
should be List< String > lines = new ArrayList<>(Files.readAllLines(tempLogFile.toPath()));
diamonds type in new ArrayList<>();